static-site/content/blog/2024-12-10-auspice-2024.md

```diff
@@ -0,0 +1,121 @@
+git tag --list --format='%(creatordate:short)' | grep '^2024' | wc -l
+-->
+
+## Table of contents
+
+- [Focus on Selected](#focus-on-selected)
+- [View in Other Platforms](#view-in-other-platforms)
+- [Performance improvements](#performance-improvements)
+- [Conveying confidence of tips through saturation (version 2.56)](#conveying-confidence-of-tips-through-saturation-version-256)
+- [Filtering on all metadata](#filtering-on-all-metadata)
+- [Full changelog](#full-changelog)
+
+## Focus on Selected
+
+When viewing large phylogenetic trees with a sparse filter, it can be difficult
+to really see and understand the selected tips as they occupy a minority of the
+vertical space available. Enabling **Focus on Selected** allows the selected
+tips to take up more vertical space compared to other tips. For example,
+[filtering the SARS-CoV-2 global 6m view to one of the earliest clades, 20I
+(Alpha)](https://nextstrain.org/ncov/gisaid/global/6m@2024-12-10?d=tree&f_clade_membership=20I%20%28Alpha%29&p=full):
+
+[![Focus on Selected example](/blog/img/focus-on-selected-example.png)]()
+
+You can toggle this focus on/off in the sidebar:
+
+[![Focus on Selected toggle](/blog/img/focus-on-selected-toggle.png)]()
+
+We [plan](https://github.com/nextstrain/auspice/issues/1368) to continue
+exploring better ways to understand sets of sequences in the context of the
+larger tree, initially by better use of the horizontal axis which should help
+for early clades in the tree such as 20I (Alpha).
+
+## View in Other Platforms
+
+Interoperability between platforms is something we've been big proponents of,
+and are part of [PH4GE](https://pha4ge.org/) to help support this. For many
+years, Nextstrain.org has supported [fetching](https://nextstrain.org/fetch)
+HTTPS-addressable Auspice JSONs.
+[UShER](https://genome.ucsc.edu/cgi-bin/hgPhyloPlace) uses this feature to link
+to Auspice views for subtrees.
+
+Other platforms have developed a similar approach to access Auspice datasets,
+often using our RESTful API. When viewing datasets in Nextstrain, you can switch
+to supported platforms via the new **VIEW IN OTHER PLATFORMS** button at the
+bottom of the page:
+
+[![View in Other Platforms button](/blog/img/view-in-other-platforms.png)]()
+
+There are currently three supported platforms:
+
+- [Taxonium](https://taxonium.org) (added in v2.53) – scales well to large
+  datasets.
+
+- [MicrobeTrace](https://microbetrace.cdc.gov) by US CDC (added in v2.53)
+
+- [Nextclade](https://clades.nextstrain.org/) (added in v2.54) – allows you to
+  add your own sequences (in-browser) to the current tree. Certain requirements
+  must be met for this to work as expected, such as the root-sequence being
+  defined for the dataset. You may get more reliable results using the manually
+  curated reference Nextclade datasets, so please double check results against
+  those.
+
+  [![View in Nextclade preview](/blog/img/view-in-nextclade.png)]()
```

Comment:
This image kind of feels like it comes out of nowhere. I think a figure caption explaining how you can go from the cattle-outbreak Nextstrain dataset to Nextclade in order to analyse your data is needed.

Reply:
How's this as an image?

![View in Nextclade preview](https://github.com/nextstrain/nextstrain.org/blob/85553e7828d8044082a444ee19695638473ec4b4/static-site/public/blog/img/view-in-nextclade.png?raw=true)

Updated in 85553e7828d8044082a444ee19695638473ec4b4